src/renderer/components/message/SelectedMessagesActions.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+
+import { useTranslationFunction } from '../../contexts'
+import Button from '../ui/Button'
```

Comment:
I'd be up for not introducing any sub-folders like `ui` for components and rather keep them all on the same level. With a project of this size one searches anyhow more than looking through file-trees, and if one does so than I'm sure sometimes you'd expect things in `ui` which are not there or the other way around.